src/time_steppers.jl

```diff
@@ -149,14 +149,14 @@ function calculate_interior_source_terms!(grid::Grid, constants, eos, cfg, u, v,
             @loop for i in (1:grid.Nx; (blockIdx().x - 1) * blockDim().x + threadIdx().x)
                 # u-momentum equation
                 @inbounds Gu[i, j, k] = (-u∇u(grid, u, v, w, i, j, k)
-                                            + fCor*avg_xy(grid, v, i, j, k)
+                                            + gU_cori(grid, v, fCor, i, j, k)
```

Comment:
This function could be named `f×u`, which is closer to the math being executed. Likewise for `f×v`. I'd also note that we seem to be using lowercase for velocity fields and upper case for the source terms (so I guess `Gu_coriolis` would be more consistent with the current naming).

Reply:
I guess to be truly similar to the math you'd need something more like `f×u_x` and `f×u_y`.

I feel that `Gu_coriolis` and `Gv_coriolis` are also clear. I'm not sure why we are writing `fCor` instead of just `f`, but if we want to we should probably keep the code neat by always writing one of `Cor`, `Cori`, or `coriolis`, rather than some mishmash between them.

These functions may also prove useful if we ever support non-traditional Coriolis terms (which may actually be important in weakly-stratified scenarios in some places).

Reply:
By the way, I'm happy to make these name changes if you add me as a collaborator on your fork.